pyrobot/plugins/admin.py

```diff
@@ -0,0 +1,435 @@
+
+import asyncio
+import logging
+import math
+import os
+from asyncio import sleep
+from datetime import datetime
+from os import remove
+from time import sleep, time
```

Comment:
don't do this...

Reply:
why

Reply:
also that import is not used anywhere, right? Try removing the unused imports, 

and try making different commands into a seperate file. 

Maybe, admin, directory and inside that different files. 🤔😬😬

Reply:
okay i will do it. thanks.